### DISCUSSION TITLE
How to deal with prerender - error on bigger projects

### DISCUSSION THREAD

**Discussion by dohomi** [vercel/next.js#14729] [Help] [answered]
Hi Guys,

I run in a pretty big project into a https://github.com/vercel/next.js/blob/master/errors/prerender-error.md due to broken/misspelled links deep nested in my page tree (headless CMS).

Due to the error the build process (SSG) never succeeds. Is there a way to print out the broken links and on which they happen? Currently its quite difficult to keep track on broken links. 

Is there (probably out of NextJS) a good spider/crawler which I could run over an entire site and test if all links are still working? Any suggestions are welcome! Thanks

**Comment by dohomi** [comment, marked as answer]
I am fighting with this error message in particular:
```
  
Error occurred prerendering page "/". Read more: https://err.sh/next.js/prerender-error  
Error: The provided export path '/' doesn't match the '/[...index]' page.
```
Is there a possibility to get more information about this kind of error? Or to still finalize the build and not stopping it?

**Reply by Timer** [reply]
This means you're returning `/` in `getStaticPaths` within the `/[...index]` page, which isn't a valid route for `/[...index]`.

**Reply by dohomi** [reply]
@Timer my issue is to find the page on which this happens. It seems to happen on one page of 800 and its hard to find this

**Reply by Timer** [reply]
The error states it's the `/[...index]` page

**Reply by dohomi** [reply]
yes, but the /[...index] page is the catch all page of all my pages, which are in total 800. It would be great if the error would not only state the wrong Link, but also on which url/slug it is getting thrown.

**Reply by Timer** [reply]
The `/[...index]` pages [does **not** match `/`](https://nextjs.org/docs/routing/dynamic-routes#catch-all-routes), which is causing this error (re: the error message).

**Reply by dohomi** [reply]
But I will definitely try to prevent this falsy entry code wise, I just wanted to share that it might be helpful if the error would be more specific on which slug it actually is being thrown

**Reply by dohomi** [reply]
Maybe I have some code from the optional catch all still inside of my codebase. As far I could see you are publishing it with the upcoming version. Thanks for your input I will investigate. Maybe Ill try the latest canary and rewrite [...index] to [[...index]]

**Reply by Timer** [reply]
@dohomi the error is very specific that you are trying to render "/" (that is the slug / path) on the page `/[...index]` which is not valid. Could you please elaborate what you mean?

The only way to get `/` to work without being a real page is optional catch all routes.

**Reply by dohomi** [reply]
Yes I meant the optional catch-all routes, I'll give it a try. Thanks for your input I'll know now how to proceed
